.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "src/plugins/intel_cpu/thirdparty/onednn"]
 	path = src/plugins/intel_cpu/thirdparty/onednn
-	url = https://github.com/openvinotoolkit/oneDNN.git
+	url = https://github.com/avoskoboinyk-lohika/oneDNN.git
```

Comment:
Just curious, do we really need to change the submodule url for the oneDNN?

Reply:
It's a temporary change, for CI

Reply:
Ah, okay, sorry

Reply:
Please do not forget to remove this before the merge.

Reply:
I removed the commit during the last master rebase.